packages/frontend/src/components/UserSettings/ProjectManagementPanel/index.tsx

```diff
@@ -162,6 +188,36 @@ const ProjectManagementPanel: FC = () => {
                     projectUuid={deletingProjectUuid}
                 />
             ) : null}
+            {isCreatePreviewOpen && lastProject && (
+                <Modal
+                    opened={isCreatePreviewOpen}
+                    onClose={() => setIsCreatePreview(false)}
+                    title={`Create preview from ${lastProject.name}`}
+                >
+                    <Text>
+                        This will create a preview project from $
+                        {lastProject.name}. The new project will have the same
+                        connections and credentials.
+                        <Group mt="sm" mb="sm">
+                            {' '}
+                            Preview name
+                            {/* TODO use inputWrapper*/}
+                            <Input value={`Preview of ${lastProject.name}`} />
+                        </Group>
+                        <Button
+                            onClick={() => {
+                                // create preview project
+                                createPreviewProject({
+                                    projectUuid: lastProject.projectUuid,
```

Comment:
this is the same as `lastProjectUuid`  in L114 correct?

Reply:
kind of, that constant is `const lastProjectUuid: string | null | undefined` which means I'd have to check it is not undefined, I guess this is easier.